app/src/main/java/org/gdg/frisbee/android/app/GoogleApiClientFactory.java

```diff
@@ -0,0 +1,37 @@
+package org.gdg.frisbee.android.app;
+
+import android.content.Context;
+
+import com.google.android.gms.appindexing.AppIndex;
+import com.google.android.gms.common.api.GoogleApiClient;
+import com.google.android.gms.drive.Drive;
+import com.google.android.gms.games.Games;
+import com.google.android.gms.plus.Plus;
+
+import org.gdg.frisbee.android.utils.PrefUtils;
+
+public final class GoogleApiClientFactory {
+    private GoogleApiClientFactory() { }
```

Comment:
In Utility classes, private constructor is always at the top in all classes in Frisbee. So if you don't count that one the rest is all static :) We can definitely discuss this, private constructor can be moved to bottom.